packages/legend-application-studio/src/stores/editor-state/element-editor-state/ClassState.ts

```diff
@@ -126,7 +126,7 @@ export class DerivedPropertyState extends LambdaEditorState {
             ? this.extractLambdaString(grammarText)
             : '',
         );
-        this.clearErrors();
```

Comment:
Isnt this change too broad. We call `convertLambdaObjectToGrammarString` when typing so would this change not delete errors when a user fixes an error ? 
Also could we update the video to show clicking the expand button

Reply:
do you think if it makes sense to add a flag to this function which tells us if user is trying to expand the lambda editor and clearErrorSourceINformation in that case?